packages/k8s/k3s/collection.yaml

```diff
@@ -1,7 +1,7 @@
 packages:
   - name: k3s-openrc
     category: k8s
-    version: "1.29.0+2"
```

Comment:
why not +3? in the end it doesn't really matter but I think ok to keep different build versions for openrc and systemd and I did some changes to the pipeline so it can handle them

Reply:
OCD basically, I could not stand them being in a different revision